### DISCUSSION TITLE
Work of he script ended with an error

### DISCUSSION THREAD

**Discussion by Doroshenko-T** [angristan/wireguard-install#419] [Help]
Hi,

Work of he script ended with an this error:

Job for wg-quick@wg0.service failed because the control process exited with error code.

OS: Debian 10.6 / 64 bit

**Comment by randshell** [comment]
First thing try to reboot, then check again the output of `systemctl status wg-quick@wg0.service` and `sudo modprobe wireguard`. If it still doesn't work, can you please paste it here for further help?

**Comment by Niemand112233** [comment]
I have the same error.

root@raspberrypi:/home/user# systemctl status wg-quick@wg0
Unit wg-quick@wg0.service could not be found.
root@raspberrypi:/home/user# modprobe wireguard
modprobe: FATAL: Module wireguard not found in directory /lib/modules/5.4.72-v7+

I rebooted already

**Comment by randshell** [comment]
@Niemand112233 raspberry-pi isn't supported at the moment #73
